src/screens/collection/collection.tsx

```diff
@@ -0,0 +1,72 @@
+import { isNonEmptyArray } from '@apollo/client/utilities';
+import { RouteProp, useRoute } from '@react-navigation/native';
+import React, { useCallback } from 'react';
+import { View, ListRenderItem, ViewToken } from 'react-native';
+import { FlatList } from 'react-native-gesture-handler';
+
+import { DataPlaceholder } from 'src/components/data-placeholder/data-placeholder';
+import { useCollectibleByCollectionInfo } from 'src/hooks/use-collectibles-by-collection.hook';
+import { useInnerScreenProgress } from 'src/hooks/use-inner-screen-progress';
+import { ScreensEnum, ScreensParamList } from 'src/navigator/enums/screens.enum';
+import { useSelectedAccountSelector } from 'src/store/wallet/wallet-selectors';
+import { formatSize } from 'src/styles/format-size';
+import { TokenInterface, emptyToken } from 'src/token/interfaces/token.interface';
+import { isDefined } from 'src/utils/is-defined';
+
+import { useCollectionStyles } from './collection.styles';
+import { CollectibleItem } from './components/collectible-item';
+
+const COLLECTIBLE_SIZE = 327;
+
+export const Collection = () => {
+  const styles = useCollectionStyles();
+  const selectedAccount = useSelectedAccountSelector();
+  const { params } = useRoute<RouteProp<ScreensParamList, ScreensEnum.Collection>>();
+
+  const collectibles = useCollectibleByCollectionInfo(
+    params.collectionContract,
+    selectedAccount.publicKeyHash,
+    params.type
+  );
+
+  const { setInnerScreenIndex } = useInnerScreenProgress(collectibles.length);
+
+  const handleChanged = useCallback((info: { viewableItems: ViewToken[] }) => {
+    if (isNonEmptyArray(info.viewableItems) && isDefined(info.viewableItems[0].index)) {
+      if (info.viewableItems[0].index === 0) {
```

Comment:
Magic zero index. What does it mean?

Reply:
at the initial position, we need to show 1 as the first item in our collection
but we have dummy component in flatlist, for left margin, so we need to figure out numeration this way